parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -244,6 +271,8 @@ pub mod pallet {
 				.map_err(|_| <Error<T>>::BLSPreparePublicKeysFailed)?;
 			<CurrentSyncCommittee<T>>::set(sync_committee_prepared);
 			<NextSyncCommittee<T>>::kill();
+			InitialCheckpointRoot::<T>::set(header_root);
+			<LatestExecutionState<T>>::kill();
```

Comment:
kill `LatestExecutionState` so that we can start syncing from the new checkpoint and no need to sync missing `ExecutionHeader` between.